src/App.tsx

```diff
@@ -52,7 +62,26 @@ function App() {
           as="textarea"
           value={textarea}
           placeholder={defaultText}
+          ref={inputRef}
+          onKeyDown={(e: any)=> {
+            if (e.keyCode === 9) {
```

Comment:
`keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), prefer to use `key` instead